### PR TITLE
logic error for remove one element from a set

### PR DESCRIPTION
**Pull request by fengprofile**

### CODE REVIEW

**Comment by dexX7** [comment]
Hi @fengprofile, thanks for submitting! This was already fixed via https://github.com/OmniLayer/omnicore/pull/593

**Comment by dexX7** [comment]
Thanks a lot for submitting this!

Can you please merge it against the `develop` branch?

**Comment by fengprofile** [comment]
Sorry, I use git not very mture. I will try it late.

File: src/omnicore/tx.cpp [hunk 1]
```diff
@@ -2009,6 +2009,11 @@ int CMPTransaction::logicMath_RevokeTokens()
         PrintToLog("%s(): rejected: property %d is not managed\n", __func__, property);
         return (PKT_ERROR_TOKENS -42);
     }
+    
+    if (sender != sp.issuer) {
```
Comment (dexX7) [inline src/omnicore/tx.cpp]: This change breaks consensus.

**Comment by dexX7** [comment]
I resubmited this pull request via #690 against the develop branch. It's going to be included in the next release.

**Pull request closed without merging**